### ISSUE TITLE
Large assessments cause the logs to wrap multiple times in a single assessment.

### ISSUE THREAD

**Issue by ATRG** [IBM/transformation-advisor-sdk#80]
**Describe the bug**
The IssueRuleProcessor logging the entire assessment unit once for every single rule can cause large assessments to wrap the logs multiple times, hiding potentially useful information.

In this particular case, I have an assessment with 12 assessment units. Each assessment unit leads to a json file over 400,000 characters long. It's an ace assessment, which currently has 35 rules. I got to watch the ta_sdk logs wrap through at least 8 times (I saw windows refresh as files were overwritten 8 times, there may have been more I missed), and ta_util seems to have wrapped at least once as well as I can no longer see the beginning of the assessment.

**To Reproduce**
1. Create a large item to be inspected, that will create a very large assessment unit json file, in combination with a middleware that has a large amount of rules.
2. Run the TA against this artifact, and watch the logs wrap.

**Expected behavior**
Logs not wrapping, assessment unit json printed at most once.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Ubuntu and Windows
 - Java version: [e.g. IBM JDK 1.8, OpenJDK 1.9]
 - TA-SDK Version 0.5.2-2

**Additional context**

The offending line in ta_sdk is: `INFO  com.ibm.ta.sdk.core.assessment.IssueRuleProcessor - queryInput:` - printing out 400k characters, for each assessment unit, for each rule.
The offending lines in ta_util are:
```
2020-07-30 17:40:24 DEBUG CompiledPath:47 - Evaluating path: @['id']
2020-07-30 17:40:24 DEBUG PredicateContextImpl:46 - Using cached result for root path: $['evaluatedConfYaml']['Defaults']['defaultApplication']
```
which repeat _a lot_ - I can only assume that it the whole assessment unit tree is being searched, and it is being printed once for every single json object that is checked.

**Comment by transformationAdvisor-sdk** [comment]
I delete the log inside the for loop in the IssueRuleProcessor class.  and reduce some log level to trace level.  By default the SDK log level is debug,  so these message will not in the log files.  It will be only logged when the log level is set to trace.

**Comment by transformationAdvisor-sdk** [comment]
fix was delivered in both 0.5.5 and 0.6.0 release.
